onnxruntime/core/session/provider_registration.cc

```diff
@@ -81,6 +84,41 @@ ORT_API_STATUS_IMPL(OrtApis::SessionOptionsAppendExecutionProvider,
                     _In_reads_(num_keys) const char* const* provider_options_values,
                     _In_ size_t num_keys) {
   API_IMPL_BEGIN
+  enum class EpID {
+    INVALID = 0,
+    DML,
+    QNN,
+    OpenVINO,
+    SNPE,
```

Comment:
is SNPE still enabled? I thought that was replaced by QNN

Reply:
QNN EP replaces SNPE. I removed mention of SNPE from the `SessionOptionsAppendExecutionProvider` C API function documentation. However, I did not remove SNPE from the implementation (left a TODO to remove SNPE entirely).